module/platform/darwin/HardwareSimulator/src/HardwareSimulator.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include <Eigen/Core>
 #include <Eigen/Geometry>
+#include <iostream>
```

Comment:
?

Reply:
ahh yes, the include for the classic print statements I used to validate my findings :joy: